src/main/java/banking/account/policy/NormalInterestCalculator.java

```diff
@@ -0,0 +1,28 @@
+package banking.account.policy;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.NavigableMap;
+import java.util.TreeMap;
+
+public class NormalInterestCalculator implements  InterestCalculator{
+
+    private final NavigableMap<BigDecimal, BigDecimal> interestRates = new TreeMap<>();
+
+    @Override
+    public BigDecimal getInterest(BigDecimal balance) {
+        calculator();
+        BigDecimal rate = (interestRates != null) ? interestRates.floorEntry(balance).getValue(): BigDecimal.ZERO;
+        return balance.multiply(rate).setScale(0, RoundingMode.DOWN);
+    }
+
+    public void calculator() {
```

Comment:
인터페이스에서 선언된 메소드 외의 메소드를 public으로 선언하여 외부에서 접근을 열어두는 것은 좋지 않습니다
```suggestion
    private void calculator() {
```